### PR TITLE
Splash screen beta img

### PR DESCRIPTION
**Pull request by mateuszpmroz**

### CODE REVIEW

File: android/app/src/main/java/io/goldwallet/wallet/MainActivity.java [hunk 1]
```diff
@@ -8,6 +8,14 @@
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
+
+        // TODO: Add source using env var
+        // String splashSrc = "@mipmap/splash"
+        // if (BuildConfig.IS_BETA === "1") {
```
Comment (KedziaPawel) [inline android/app/src/main/java/io/goldwallet/wallet/MainActivity.java:14]: comments

File: src/helpers/images.ts [hunk 2]
```diff
@@ -0,0 +1,6 @@
+import Config from 'react-native-config';
```
Comment (KedziaPawel) [inline src/helpers/images.ts:1]: we have root/config/index.js, all config things go there

File: src/helpers/images.ts [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+
+import { images } from 'app/assets';
+
+export const getLogoSource = () =>
+  Config.IS_BETA === '1' ? images.goldWalletLogoBlackBeta : images.goldWalletLogoBlack;
```
Comment (KedziaPawel) [inline src/helpers/images.ts:6]: IS_BETA should be false/true not a number, parse the string from env like `isBeta: JSON.parse(IS_BETA)` in the config/index.js

File: src/navigators/Navigator.tsx [hunk 4]
```diff
@@ -1,11 +1,13 @@
 import { NavigationContainer } from '@react-navigation/native';
 import JailMonkey from 'jail-monkey';
 import React from 'react';
+import Config from 'react-native-config';
```
Comment (KedziaPawel) [inline src/navigators/Navigator.tsx:4]: same

File: src/helpers/images.ts [hunk 5]
```diff
@@ -0,0 +1,6 @@
+import Config from 'react-native-config';
+
+import { images } from 'app/assets';
+
+export const getLogoSource = () =>
```
Comment (KedziaPawel) [inline src/helpers/images.ts:5]: `export const logoSource = isBeta ? images.goldWalletLogoBlackBeta : images.goldWalletLogoBlack;`

it doesn't have to be a function, the variable would have better performance as it won't have to be invoked every time the component will be rerendered

File: .env.test [hunk 6]
```diff
@@ -3,4 +3,5 @@ BTCV_NETWORK=bitcoinvault
 HOST=e1.electrumx.bitcoinvault.global
```
Comment (KedziaPawel) [inline .env.test:3]: add also to .env.example

**Review by KedziaPawel — COMMENTED** [review]
did you compress images with a tool like https://compressor.io/ ?

**Comment by mateuszpmroz** [comment]
> did you compress images with a tool like https://compressor.io/ ?

No, I've copied them directly without compressing them

File: src/helpers/images.ts (hunk 3 above)
Comment (mateuszpmroz) [reply src/helpers/images.ts:6]: Okay, good that something like this exists, didn't notice

**Comment by mateuszpmroz** [comment]
> > did you compress images with a tool like https://compressor.io/ ?
> 
> No, I've copied them directly without compressing them

@KedziaPawel Right now files are compressed

**Review by KedziaPawel — APPROVED** [review]

**Pull request merged**